packages/moonstone/VideoPlayer/Video.js

```diff
@@ -0,0 +1,297 @@
+import {forward} from '@enact/core/handle';
+import ForwardRef from '@enact/ui/ForwardRef';
+import {Media, getKeyFromSource} from '@enact/ui/Media';
+import Slottable from '@enact/ui/Slottable';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import css from './VideoPlayer.less';
+
+import PropTypes from 'prop-types';
+
+/**
+ * Adds support for preloading a video source for `VideoPlayer`.
+ *
+ * @class VideoBase
+ * @memberof moonstone/VideoPlayer
+ * @ui
+ * @private
```

Comment:
Shouldn't be `@private`. It's a component for all the people. 👐

Reply:
Actually, I remember now! `VideoBase` is set to `@private` since it is only exported from the submodule. `Video` (with `Slottable` applied) is marked as `@public` and exported from `moonstone/VideoPlayer`.

I did update the docs for the public export to illustrate the advanced features the component enables.

Reply:
Gotcha. One more thing. Let's add the `@exports Video` at the top of `VideoPlayer.js`